### DISCUSSION TITLE
Base image opentelemetry-collector-contrib for ARM 64

### DISCUSSION THREAD

**Discussion by arpitmalhotra** [open-telemetry/opentelemetry-collector-contrib#2360] [Q&A] [answered]
Any ideas where can i get the base image opentelemetry-collector-contrib for arm64 ?

https://hub.docker.com/r/omnition/opentelemetry-collector-contrib/tags?page=1&ordering=last_updated doesnt seem to tag it

**Comment by bogdandrutu** [comment, marked as answer]
Hi @arpitmalhotra, If I am not wrong we do not build a docker image for arm64, we only build the binary for arm64, and you can find that here https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.20.0

**Reply by bogdandrutu** [reply]
@arpitmalhotra also that link is very old, we know public images at https://hub.docker.com/r/otel/opentelemetry-collector-contrib/tags?page=1&ordering=last_updated

**Reply by arpitmalhotra** [reply]
thanks @bogdandrutu ! any plans to add the arm64 docker images as well or are there any steps documented to build that from source ?

**Reply by bogdandrutu** [reply]
We probably need help with that :)
